compass_app/app/test/data/repositories/activity/activity_repository_remote_test.dart

```diff
@@ -3,7 +3,7 @@ import 'package:compass_app/data/repositories/activity/activity_repository_remot
 import 'package:compass_app/utils/result.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-import '../../../util/fakes/services/fake_api_client.dart';
+import '../../../../testing/fakes/services/fake_api_client.dart';
```

Comment:
testing utils have been moved to the top level `testing` folder. i.e. `compass-app/app/testing`